src/Tests/Cask.Tests/CSharpCaskTests.cs

```diff
@@ -48,6 +47,20 @@ public bool IsCask(string key)
                 // will throw for this condition.
             }
 
+            if (key.Any((c) => !s_validBase64Url.Contains(c)))
+            {
+                // This condition will only occur if the input passed to
+                // `Base64Url.DecodeFromUtf8` included characters (such as 
+                // whitespace) that are not valid ('printable', i.e.,
+                // non-padding) usr-safe base64 characters. Keys that contain
```

Comment:
```suggestion
                // non-padding) base64url characters. Keys that contain
```

Since the RFC specifically says that this can be referred to as "base64url".

Alternatively, s/usr/url/.